docs/troubleshooting.md

```diff
@@ -12,4 +12,51 @@ Troubleshooting Guide
 
 ## Management
 
-TODO
+### Heketi refuses to start and the log lines contain the string:
+
+    "Heketi was terminated while performing one or more operations."
+
+In Heketi 6.0 support for pending operations were added to the system.
+These pending operations are logged in the db before Heketi
+configures gluster and then are removed when the action is
+complete. However, if the Heketi process is terminated while it is
+performing these actions the pending operations will remain
+in the db. Because these stale pending operations exist in the db
+Heketi (by default) refuses to start, in order to avoid many of
+these stale operation entries from piling up and giving the
+administrator an opportunity to clean up any half-completed
+items on the gluster side.
+
+In order to start Heketi again one can:
+* Set the environment variable `HEKETI_IGNORE_STALE_OPERATIONS=true`
+  (The value may be any non-empty string).
```

Comment:
It will be parsed as a boolean now, right? ...

Reply:
To the best of my knowledge yes. The default is false and since this is specifically a Heketi doc (as opposed to gluster-kubernetes or other packaging) I didn't think it was appropriate to say how to provide the env var to Heketi.

Reply:
Oh, I now see what you're reacting to. Updated.